bin/api-docs/gen-components-docs/markdown/index.mjs

```diff
@@ -49,5 +46,5 @@ export function generateMarkdownDocs( { typeDocs, subcomponentTypeDocs } ) {
 
 	return json2md(
 		[ ...mainDocsJson, ...subcomponentDocsJson ].filter( Boolean )
-	);
+	).replace( /\n+$/gm, '\n' ); // clean unnecessary consecutive newlines
```

Comment:
Noting that this cleans up only trailing ones, maybe we want to update the comment to reflect that?

Reply:
[This](https://regex101.com/r/YHxZ1L/2) is a `global` + `multiline` regex, intended to remove all excessive newlines within in the entire readme output.